kratos/geometries/plane.h

```diff
@@ -60,7 +60,9 @@ class Plane
     {
         array_1d<double, 3> edge1 = p1 - p0;
         array_1d<double, 3> edge2 = p2 - p0;
-        mNormal   =  MathUtils<double>::UnitCrossProduct(edge1,edge2);
+
+        MathUtils<double>::UnitCrossProduct(mNormal, edge2, edge1);
```

Comment:
I would not change the order here as it will continue to be in the wrong direction. This will need a BugFix PR

Reply:
Agree, but that means we are changing the way Plane work (assuming right now everyone is using the inverted result), I will put an issue for that.